modules/http4s-kernel/src/smithy4s/http4s/kernel/package.scala

```diff
@@ -37,7 +37,7 @@ import fs2.Chunk
 // scalafmt: { maxColumn = 120}
 package object kernel {
 
-  private[smithy4s] def toSmithy4sHttpRequest[F[_]: Concurrent](req: Request[F]): F[Smithy4sHttpRequest[Blob]] = {
+  def toSmithy4sHttpRequest[F[_]: Concurrent](req: Request[F]): F[Smithy4sHttpRequest[Blob]] = {
```

Comment:
**suggestion (if-minor, non-blocking):** perhaps all these functions should go to some kind of `conversions` object, keeping them in the package object seems a bit strange.

Reply:
FYI this is a really minor thing that could be resolved in 0.19 ;P